index.html

```diff
@@ -21,31 +21,50 @@
     <main>
       <h1>COLORANDOM</h1>
       <section class="color-boxes">
-        <div>
+        <div id="color1">
           <div class="swatch" id="box1"></div>
-          <p>
-            #111111<span class="material-symbols-outlined"> lock_open </span>
-          </p>
+            <div class="box-details">
+              <p id="hex1">
+                #111111
+              </p>
+              <span class="material-symbols-outlined" id="lock1"> lock_open </span>
+            </div>
         </div>
-        <div>
+        <div id="color2">
           <div class="swatch" id="box2"></div>
-          <p>#222222<span class="material-symbols-outlined"> lock </span></p>
+            <div class="box-details">
+              <p id="hex2">
+                #222222
+              </p>
+              <span class="material-symbols-outlined" id="lock2"> lock </span>
+            </div>
         </div>
-        <div>
+        <div id="color3">
           <div class="swatch" id="box3"></div>
-          <p>
-            #333333<span class="material-symbols-outlined"> lock_open </span>
-          </p>
+            <div class="box-details">
+              <p id="hex3">
+                #333333 
+              </p>
+              <span class="material-symbols-outlined" id="lock3"> lock_open </span>
```

Comment:
Whenever trying to change the hex code through Javascript, the` innerText` was returning the hex code along with the text for the lock. By removing the `span` element outside of `<p>', `hex1.innerText` would return only the hex code.